src/spetlr/delta/delta_handle.py

```diff
@@ -25,22 +28,49 @@ class DeltaHandleInvalidFormat(DeltaHandleException):
 
 
 class DeltaHandle(TableHandle):
-    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+    def __init__(
+        self,
+        name: str,
+        location: str = None,
+        data_format: str = "delta",
+        options_dict: dict = None,
+        ignore_changes: bool = True,
+        stream_start: datetime = None,
```

Comment:
having constructor arguments that are not basic types (string, number) is actually quite problematic here. The configurator does not currently support (although we could make it support) keys whose values are dict. Hence, the options_dict could not be set in the constructor based on the `from_tc` method.

I suggest that you change the type of options_dict to `Dict[str:str]`. For now, you leave this out of the from_tc mthod since it is not settable that way yet anyways. We can then make another pull request where the configurator supports such structures and improve the from_tc method.

Reply:
I have implemented your suggestions.